File: android-java/javadeneme/app/src/main/java/com/example/javadeneme/MainActivity.java
package com.example.javadeneme;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {
    public static void main(String[] args){
        StudentInfo s1 = new StudentInfo();
        s1.name ="Arif";
        s1.midScore =60;
        s1.finScore =40;
        s1.CalculateAverage(s1.midScore,s1.midScore);



    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Log.e("Etiket","Tebrikler Uygulama çalıştı");


    }
}

File: android-kotlin/Kotlin2/app/src/main/java/com/example/kotlin2/MainActivity.kt
package com.example.kotlin2

import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.ListView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        var p1 = Person("Arif",22,"Student")
        var p2 = Person("Ali",25,"Engineer")
        var p3 = Person("Veli",30,"Doctor")

        val personNameList = listOf(p1.name,p2.name,p3.name)

        val list = findViewById<ListView>(R.id.listView)

        val adapter = ArrayAdapter(this,android.R.layout.simple_list_item_1,personNameList)

        list.adapter = adapter





    }
}